bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/PageListItemImpl.java

```diff
@@ -83,6 +84,9 @@ public String getPath() {
         return page.getPath();
     }
 
+    @Override
+    @JsonIgnore
+    public String getName() { return page.getName(); }
```

Comment:
Could you expand this to multiple lines to follow the same style as the rest of the code?